.eslintrc.js

```diff
@@ -43,13 +37,36 @@ module.exports = {
         '@typescript-eslint/no-object-literal-type-assertion': 'off',
         '@typescript-eslint/no-use-before-define': 'off',
         '@typescript-eslint/explicit-module-boundary-types': 'off',
-        '@typescript-eslint/no-duplicate-imports': 'error',
+        '@typescript-eslint/no-duplicate-imports': 'warn',
+        '@typescript-eslint/array-type': ['warn', {
+            default: 'generic',
+        }],
+        '@typescript-eslint/consistent-type-assertions': 'error',
+        '@typescript-eslint/consistent-type-imports': 'error',
+        '@typescript-eslint/member-delimiter-style': 'error',
+        '@typescript-eslint/no-implicit-any-catch': 'warn',
```

Comment:
I don't think there's a point in `catch (err: unknown)`. It will always be unknown. JS doesn't support multiple catches based on error type. One may throw anything, like `throw 'message'`, but it will not generate stack trace, so people normally do `throw new Error('message')`